tsdb/db.go

```diff
@@ -597,6 +597,85 @@ func (db *DBReadOnly) Blocks() ([]BlockReader, error) {
 	return blockReaders, nil
 }
 
+// LastBlockID returns the BlockID of latest block.
+func (db *DBReadOnly) LastBlockID(logger log.Logger) (string, error) {
+	select {
+	case <-db.closed:
+		return "", ErrClosed
+	default:
```

Comment:
What is the default case here? If there is nothing then remove the case.

Reply:
In select block, we are checking whether the DB is closed,and if so, returns an error indicating that the database connection has been closed. If the channel is still open, the code block continues without blocking and executes the default case, which in this case is an empty block.

**REF:** Non-Blocking Channel Operations https://gobyexample.com/non-blocking-channel-operations